src/jit/lsra.cpp

```diff
@@ -6502,6 +6562,45 @@ regNumber LinearScan::rotateBlockStartLocation(Interval* interval, regNumber tar
 }
 #endif // DEBUG
 
+#ifdef _TARGET_ARM_
+//--------------------------------------------------------------------------------------
+// isSecondHalfReg: Test if recRec is second half of double reigster
+//                  which is assigned to an interval.
+//
+// Arguments:
+//    regRec - a register to be tested
+//    interval - an interval which is assigned to some register
+//
+// Assumptions:
+//    The interval must have been assigned to some register
```

Comment:
This assumption does not appear to be correct. If the interval is not assigned to a register, it will simply return false.